fairworkflows/cwl.py

```diff
@@ -94,14 +100,19 @@ def run_workflow(wf_path: Union[Path, str], inputs: Dict[str, any], output_dir:
     if base_dir:
         cwltool_args += ['--basedir', str(base_dir)]
 
+    cwltool_args += ['--logFile', str(log_path)]
+
     try:
         cwltoil.main(cwltool_args + [wf_path] + wf_input)
     except SystemExit as e:
         raise CWLException(f'Workflow {wf_path} with inputs {inputs} has failed.', e)
 
     _logger.debug('CWL tool has run successfully.')
 
-    return output_dir
+    with open(log_path, 'r') as infile:
+        runlogs = infile.read()
+
+    return runlogs
```

Comment:
I don't think we necessarily need to add this part, I'm assuming you put this here for debugging. At some point we need to have a discussion about what we need to have as output of a workflow run. A dict listing all output files for example.
I'm not sure if we need to have the log as output as well, or that we should keep that information away from the user.

Reply:
Hm I see now that you need this log as output to store in the RO crate. I suggest we return a tuple for now: `(OUTPUT_DIR, LOG)`

Reply:
OK, no problem, I'll make that the returned values